pkg/gui/controllers/helpers/sub_commits_helper.go

```diff
@@ -0,0 +1,62 @@
+package helpers
+
+import (
+	"github.com/jesseduffield/lazygit/pkg/commands/git_commands"
+	"github.com/jesseduffield/lazygit/pkg/commands/models"
+	"github.com/jesseduffield/lazygit/pkg/gui/types"
+)
+
+type SubCommitsHelper struct {
+	c *HelperCommon
+
+	refreshHelper *RefreshHelper
+	setSubCommits func([]*models.Commit)
+}
+
+func NewSubCommitsHelper(
+	c *HelperCommon,
+	refreshHelper *RefreshHelper,
+	setSubCommits func([]*models.Commit),
+) *SubCommitsHelper {
+	return &SubCommitsHelper{
+		c:             c,
+		refreshHelper: refreshHelper,
+		setSubCommits: setSubCommits,
+	}
+}
+
+func (self *SubCommitsHelper) ViewSubCommits(ref types.Ref, refForSymmetricDifference string, context types.Context, showBranchHeads bool) error {
```

Comment:
I'd make this an opts struct rather than use positional args given that it's getting long

Reply:
Changed in e2acae14d794.